src/paint.rs

```diff
@@ -258,10 +258,14 @@ impl<'p> Painter<'p> {
                 Painter::right_fill_background_color(&mut line, fill_style);
             } else if let Some(BgFillMethod::Spaces) = bg_fill_mode {
                 let text_width = ansi::measure_text_width(&line);
+                let mut repeat_count:usize = 0;
+                if text_width < config.available_terminal_width {
+                    repeat_count  = config.available_terminal_width - text_width;
+                }
                 line.push_str(
                     #[allow(clippy::unnecessary_to_owned)]
                     &fill_style
-                        .paint(" ".repeat(config.available_terminal_width - text_width))
+                        .paint(" ".repeat(repeat_count))
                         .to_string(),
```

Comment:
You can use [saturating_sub()](https://doc.rust-lang.org/std/primitive.i16.html#method.saturating_sub) directly there.